airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -428,31 +431,40 @@ def __init__(
         unmanaged_v_cpus: int | None = None,
         service_role: str | None = None,
         tags: dict | None = None,
-        max_retries: int | None = None,
+        poll_interval: int = 30,
+        max_retries: int = 120,
         status_retries: int | None = None,
         aws_conn_id: str | None = None,
         region_name: str | None = None,
+        deferrable: bool = True,
         **kwargs,
     ):
         super().__init__(**kwargs)
+        if status_retries is not None:
```

Comment:
Should we remove it from the list of parameters then? To not encourage people using it

Reply:
yes I can do that, then I have to look in the `kwargs` to see if I need to print the warning, and remove it before calling `super()` otherwise it triggers the "unexpected arg" error.
I didn't do it because I thought it was a lot simpler that way.

Reply:
I agree it is simpler but I prefer removing it from the parameters to not suggesting a useless parameter to users